### ISSUE TITLE
Not working on Axios

### ISSUE THREAD

**Issue by iqbalansyor1** [alexbrazier/react-native-network-logger#55]
I am using Axios for network call and using `router-flux` for the navigation. 
I have followed all the steps from README
```
import { startNetworkLogging } from 'react-native-network-logger';

startNetworkLogging();
AppRegistry.registerComponent(appName, () => App);
```
```
            <Scene
              key="networkLogger"
              component={() => <NetworkLogger theme={'dark'} />}
              back
              headerLayoutPreset="center"
              renderTitle={() => <PlainTitleBar />}
            />
```
```
      Actions.networkLogger({
        onBack: () => { Actions.pop() }
      });
```
But no request or response was shown. I have tested it using the simulator, also on a real devices with both debug and release variants. Is there any config that I missed? Thanks

<img width="309" alt="Screen Shot 2021-08-23 at 16 34 13" src="https://user-images.githubusercontent.com/75062123/130426990-7785236d-bc98-4feb-9da9-256c3b28652e.png">

**Comment by alexbrazier** [comment]
There is nothing obviously wrong with your setup from what you have posted. Are you using any other logging tools like reactotron. In this case both tools make use of the same intercepter and only one of them can run at once.

**Comment by iqbalansyor1** [comment]
Yeaahh it worked, need to remove reactotron. Thankss.

**Comment by alexbrazier** [comment]
This issue should now be fixed in [v1.11.0](https://github.com/alexbrazier/react-native-network-logger/releases/tag/v1.11.0). You'll need to start the logger with the `forceEnable: true` option. This will override any other network interceptors that are running.

```ts
startNetworkLogging({ forceEnable: true });
```
